inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -125,7 +125,7 @@ def __init__(self,
             # If an exception occurs during string conversion, raise that exception
             raise SotaError('package_list is not a string in manifest') from e
 
-        self._reboot_device = parsed_manifest['deviceReboot']        
+        self._reboot_device = parsed_manifest['deviceReboot']
```

Comment:
**<h2>$\color{#FFD700}Coding Style: CodeFormatting$</h2>**
**Suggestion:** Remove unnecessary whitespace for consistent code formatting.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
            raise SotaError('package_list is not a string in manifest') from e
            self._reboot_device = parsed_manifest['deviceReboot']
            self._install_check_service = install_check_service
```

Reply:
this is confusing old and new diff lines